docs/io/votable/index.rst

```diff
@@ -625,6 +625,9 @@ See Also
    that
 .. include:: performance.inc.rst
 
+.. include:: dataorigin.rst
+
```

Comment:
Was it your intent to bury this dataorigin page within "see also" after the performance section? Might be hard to discover.

Reply:
Actually, this is also why you are getting this warning: `WARNING: duplicate label astropy-io-votable-dataorigin_intro`

You should not use `include` directive with it. It should be under `toctree` listing somewhere.